src/stan/variational/families/normal_fullrank.hpp

```diff
@@ -251,7 +251,7 @@ namespace stan {
 
```

Comment:
@dustinvtran / @akucukelbir: can you add at least one test for the calc_grad function? I just looked and none of the tests in Stan actually test it directly.

There were only tests that it failed, not that ran.